### PR TITLE
glow not following player state fix

### PR DESCRIPTION
**Pull request by buyBread**

### CODE REVIEW

File: data/menu/nullifiedcat/visuals/glow.xml [hunk 1]
```diff
@@ -11,7 +11,6 @@
                     <Option name="Not visible" value="2"/>
                 </Select>
             </LabeledObject>
-            <AutoVariable width="fill" target="glow.self-rainbow" label="Self Rainbow"/>
```
Comment (BenCat07) [inline data/menu/nullifiedcat/visuals/glow.xml]: You can't just remove that

File: data/menu/nullifiedcat/visuals/chams.xml [hunk 2]
```diff
@@ -8,7 +8,6 @@
             <AutoVariable width="fill" target="chams.health" label="Health"/>
             <AutoVariable width="fill" target="chams.recursive" label="Recursive"/>
             <AutoVariable width="fill" target="chams.single-pass" label="Single pass"/>
-            <AutoVariable width="fill" target="chams.self-rainbow" label="Self Rainbow"/>
```
Comment (BenCat07) [inline data/menu/nullifiedcat/visuals/chams.xml]: You can't just remove that

File: src/visual/EffectChams.cpp [hunk 3]
```diff
@@ -27,7 +27,6 @@ static settings::Boolean weapons_white{ "chams.white-weapons", "true" };
 static settings::Boolean legit{ "chams.legit", "false" };
 static settings::Boolean singlepass{ "chams.single-pass", "false" };
 static settings::Boolean chamsself{ "chams.self", "true" };
-static settings::Boolean rainbow{ "chams.self-rainbow", "true" };
```
Comment (BenCat07) [inline src/visual/EffectChams.cpp]: Why remove this?

Comment (buyBread) [reply src/visual/EffectChams.cpp]: It doesn't do anything. Chams follow the player state, so I assumed you guys intended it to work like that.

File: data/menu/nullifiedcat/visuals/glow.xml (hunk 1 above)
Comment (buyBread) [reply data/menu/nullifiedcat/visuals/glow.xml]: It doesn't do anything

File: data/menu/nullifiedcat/visuals/chams.xml (hunk 2 above)
Comment (buyBread) [reply data/menu/nullifiedcat/visuals/chams.xml]: It doesn't do anything

**Comment by buyBread** [comment]
```cpp
        if (ent->m_IDX == LOCAL_E->m_IDX)
            if (LOCAL_E->m_iTeam() == TEAM_BLU)
                return colors::blu;
            else
                return colors::red;
```
Removing this does fix it.

**Comment by BenCat07** [comment]
Yes. but if you want to make a pull request with changes to cathook, it shouldn't remove unrelated features. We can't merge your pull request in this state, so we will have to either make a fork with your fixing changes, or you simply remove the breaking ones from your fork

**Comment by BenCat07** [comment]
i'm mad, chams/glow rainbow are unused for no reason

**Comment by BenCat07** [comment]
We have decided, self rainbow is not needed due to cat state

**Pull request merged**